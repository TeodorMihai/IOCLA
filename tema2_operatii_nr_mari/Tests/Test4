85FB7813 + -56FA
