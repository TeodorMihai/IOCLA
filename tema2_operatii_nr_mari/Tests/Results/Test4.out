1378FB85
FA56
1921FB85
85FB2119
